nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core/src/main/java/org/apache/nifi/controller/repository/FileSystemRepository.java

```diff
@@ -404,24 +404,13 @@ public FileVisitResult visitFile(final Path file, final BasicFileAttributes attr
     @Override
     public long getContainerCapacity(final String containerName) throws IOException {
         final Path path = containers.get(containerName);
-        if (path == null) {
```

Comment:
This null check ensure's this repository has the specified container. This check was moved to the utility method in `FileUtils`. I'm not sure there is a condition where the containerName is unrecognized but this is general defensive programming. No issue here, just referencing it for other comments.